test/tests/executors/multiple/test.i

```diff
@@ -0,0 +1,55 @@
+[Problem]
+  solve = false
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+[]
+
+[BCs]
+  [left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  []
+  [right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  []
+[]
+
+[Executors]
+  [my_first_executor]
+    type = BinaryTestExecutor
+    inner1 = i1
+    inner2 = i2
```

Comment:
Could we do ` inner = 'i1 i2'`?

Reply:
If you write an executor to do that sure.

Reply:
This is just a test object... but we didn't add "vector" capability for this system yet.